### ISSUE TITLE
Fatal Errors on pre-set commands

### ISSUE THREAD

**Issue by FTCTeam4081** [ftctechnh/ftc_app#122]
So when we last downloaded the App master and imported it into Android Studio we got a bunch of fatal errors. They were all on the pre-set methods that control motors. 
![fatal erro](https://cloud.githubusercontent.com/assets/13933347/12364863/2bdd597e-bb9f-11e5-91dc-093679f94a9b.png)

![fatal error2](https://cloud.githubusercontent.com/assets/13933347/12364800/cd4bb95a-bb9e-11e5-9b17-594e76f03f57.png)

The photos show some examples. We tried going back and using an older version, which worked fine. Anybody have any ideas what's going on?

**Comment by smh3005** [comment]
Have you imported everything correctly?

Did you install the SDK in the right folder?

**Comment by calebsander** [comment]
If rebuilding doesn't work, this is always fixed for me by going to `File > Invalidate Caches/Restart`.

**Comment by FTCTeam4081** [comment]
@smh3005, yes. We imported it the same way we did the older version.
@coder-bot, we tried that, and this is what we got:
![fatal error 3](https://cloud.githubusercontent.com/assets/13933347/12680241/bdfd4788-c676-11e5-9abd-ae843967346e.png)
We tried it twice.
@calebsander, we tried that too, and we got the same errors.
We've tried this on a few different computers, and we got fatal errors on all of them.
Thanks for all the help!
